data_steward/resource_files/fields/measurement_ext.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
This is not needed for this PR but adding a note saying this cleaning rule expects `measurement_ext` table to be present in the dataset will suffice.

Reply:
Should I remove it?

Reply:
Yes, there is a different ticket that adds schemas for all the _ext tables.

Reply:
Removed and added note to `clean_height_weight.py` about `measurement_ext` table needing to be present in the dataset in order for the cleaning rule to run properly.

Reply:
I am watching this because I also have a measurement table cleaning rule that requires a measurement_ext table.  I created `curation/data_steward/resource_files/fields/extension_tables/measurement_ext.json` and modified the `fields_for(table_name)` and `cdm_schema()` methods.  I wanted to talk about it at the eng sync today.  My branch is lb/clean_implausible_measurements.